plugins/domains/d.pr.js

```diff
@@ -23,12 +20,22 @@ module.exports = {
                 // height: oembed.height
             };
         }
+        if (/mp4/i.test(oembed.variant)) {
+            var url = cheerio('video source').attr('src');
+            if (url) {
+                return {
+                    href: url,
+                    type: CONFIG.T.video_mp4,
```

Comment:
MP4s extracted this way need to be validated. Please use `accept` instead of `type` to pass it to validators:

`accept: CONFIG.T.video_mp4` or `accept: [CONFIG.T.video_mp4]`